palette/Cargo.toml

```diff
@@ -31,9 +31,11 @@ named = ["phf"]
 random = ["rand"]
 serializing = ["serde", "std"]
 find-crate = ["palette_derive/find-crate"]
-std = ["alloc", "approx?/std"]
+std = ["alloc", "approx?/std", "wide?/std", "simba?/std"]
```

Comment:
Just double checking, are all of these required? We didn't need wide's "std" feature before, for example. The same for "libm". If we can get away without some of them, that's better.